apple/internal/rule_factory.bzl

```
@@ -267,6 +267,11 @@ Do not change its value.
             doc = """
 This attribute is public as an implementation detail while we migrate the architecture of the rules.
 Do not change its value.
+    """,
+        ),
+        "signopts": attr.string_list(
```

I wonder if this should be:

```suggestion
        "codesignopts": attr.string_list(
```

?

---

done